src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -48,7 +48,15 @@ public int Create()
                 .Column<string>(nameof(OpenIdTokenIndex.Status))
                 .Column<string>(nameof(OpenIdTokenIndex.Subject)));
 
-            return 1;
+            return UpdateFrom1();
```

Comment:
No, you can just keep returning `1` and the service will run the `UpdateFrom1` automatically. But I'd actually recommend to return `2` and change this method to have the column. This way new setups don't have to run an `ALTER` command.